### PR TITLE
UI listings and profile details pages responsiveness

### PR DESCRIPTION
**Pull request by rajivtitus**
### What this PR does (required):
- Added mobile responsiveness to 'Listings' and 'Profile Details' pages
- Users can now sort profiles in the dashboard based on prices
- Fixed a bug, the first condition in the profile controller was fetching all profiles even the ones with 'isAvailable' set to false.

### Screenshots / Videos (required):
- 
- 
- 

### Any information needed to test this feature (required):
- Search using the landing page
- Once you are on dashboard you can sort profiles
- For mobile responsive UI, hit F12 and look through

### Any issues with the current functionality (optional):
- None

### CODE REVIEW

**Review by gursharan4312 — APPROVED** [review]

File: client/src/components/Listings/ProfileList/ProfileList.tsx [hunk 1]
```diff
@@ -1,35 +1,55 @@
...
   const [page, setPage] = useState<number>(1);
+  const [sortAscPrice, setSortAscPrice] = useState<boolean>(true);
+  const [sortedProfiles, setSortedProfiles] = useState<Profile[]>([]);
   const itemsPerPage = 9;
-  const noOfPages = Math.ceil(sitterProfiles.length / itemsPerPage);
+  const noOfPages = Math.ceil(sortedProfiles.length / itemsPerPage);
   const classes = useStyles();
 
+  useEffect(() => {
+    const profiles = sortByPrice(sortAscPrice);
```
Comment (rajivtitus) [inline client/src/components/Listings/ProfileList/ProfileList.tsx:25]: Hello

File: client/src/components/Listings/ProfileList/ProfileList.tsx [hunk 2]
```diff
@@ -1,35 +1,55 @@
...
+    setSortedProfiles(profiles);
+  }, [sortAscPrice, sortByPrice]);
+
```
Comment (rajivtitus) [inline client/src/components/Listings/ProfileList/ProfileList.tsx:28]: - To improve on attention to details when it comes to make sure you are outputting quality code to the main branch, test all base cases and edge cases before you merge your code in. Small changes could have a rippling effect on other features.
